docs/source/en/using-diffusers/sdxl.md

```diff
@@ -0,0 +1,354 @@
+# Stable Diffusion XL
+
+[[open-in-colab]]
```

Comment:
Does the colab work?

Reply:
Notebook should work once its generated!